package-lock.json

```diff
@@ -10813,9 +10811,9 @@
       }
     },
     "web-streams-polyfill": {
-      "version": "3.2.1",
```

Comment:
Just a heads up, can we clean out this package-lock once done just to be the changes we want only. git reset them to HEAD and pull in the privacy config.